int/tests/conftest.py

```diff
@@ -253,6 +275,7 @@ async def _make_did():
 
 @pytest.fixture(scope="session")
 async def accepted_taa(backchannel):
+    backchannel.timeout = 15
     result = (await fetch_taa.asyncio(client=backchannel)).result
```

Comment:
This is a recent discovery of mine but I think we should take advantage of the `Client.with_timeout(new_timeout)` method:

```suggestion
    result = (await fetch_taa.asyncio(client=backchannel.with_timeout(15))).result
```